R/serializer-csv.R

```diff
@@ -0,0 +1,17 @@
+#' @rdname serializers
+#' @param ... extra arguments supplied to respective internal serialization function.
```

Comment:
This line shouldn't be needed.

Reply:
I need that for custom NAs

Reply:
It is already being supplied here: https://github.com/rstudio/plumber/blob/ac96489e2b679ac8bed3379d2e5b5b4bc34583d3/R/serializer-json.R#L2

When documented, `serializer_csv` should share the same parameter information with `serializer_json` as they have the same `@rdname`.

Reply:
When documenting / checking with this line missing, does it give a warning?

Reply:
```suggestion
```

Reply:
I'm going to commit this as it's already covered in serializer-json.